mythril/laser/ethereum/call.py

```diff
@@ -0,0 +1,142 @@
+import logging
+from z3 import BitVec, simplify
+import mythril.laser.ethereum.util as util
+from mythril.laser.ethereum.state import Account
+from mythril.laser.ethereum.svm import CalldataType
+import re
+
+"""
+This module contains the business logic used by Instruction in instructions.py
+to get the necessary elements from the stack and determine the parameters for the new global state.
+"""
+
+
+def get_call_parameters(global_state, dynamic_loader, with_value=False):
+    """
+    Gets call parameters from global state
+    Pops the values from the stack and determines output parameters
+    :param global_state: state to look in
+    :param dynamic_loader: dynamic loader to use
+    :param with_value: whether to pop the value argument from the stack
+    :return: callee_account, call_data, value, call_data_type, gas
+    """
+    state = global_state.mstate
+    instr = global_state.get_current_instruction()
+
+    if with_value:
+        gas, to, value, meminstart, meminsz, memory_out_offset, memory_out_size = \
+            state.stack.pop(), state.stack.pop(), state.stack.pop(), state.stack.pop(), state.stack.pop(), state.stack.pop(), state.stack.pop()
+    else:
+        gas, to, meminstart, meminsz, memory_out_offset, memory_out_size = \
+            state.stack.pop(), state.stack.pop(), state.stack.pop(), state.stack.pop(), state.stack.pop(), state.stack.pop()
+        value = None
+
+    callee_address = get_callee_address(global_state, dynamic_loader, to)
+
+    callee_account = None
+    call_data, call_data_type = get_call_data(global_state, meminstart, meminsz, False)
+
+    if int(callee_address, 16) >= 5:
```

Comment:
when callee_address = 0 then callee_account won't get initialized, in that case that None might circulated around.I am not sure whether ppl will call that address or not

Reply:
Ah, this seems to have gotten lost in merging updates in laser.